mythril/mythril.py

```diff
@@ -118,40 +119,69 @@ def _init_mythril_dir(self):
         return mythril_dir
 
     def _init_config(self):
```

Comment:
The return parameter of this function is the leveldb dir, this is not reflected in the signature or function name, it seems to mee that the getting of the leveldb should be done in a separate function. But for now can you make the docs reflect that this happens, and create an issue to fix this if you agree with me ^^.

Reply:
i was actually thinking of separating leveldb part, setting up of the config file in a different pr.

Reply:
:+1: sure,